_content/av/transnationalizing-buddhism_jones.eric.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Transnationalizing Buddhism with Dr. John Marston"
+authors:
+  - "Eric Jones"
+subcat: podcast
+external_url: "https://soundcloud.com/seacrossroads/transnationalizing-cambodian-buddhism-with-john-marston"
+drive_links:
+  - "https://drive.google.com/file/d/1aygIZzNvzaX2FzWiQdHaAdPP9k6_u6Y9/view?usp=drive_link"
+course: cambodian
+tags:
+  - pilgrimage
+year: 2017
+month: nov
+series: Southeast Asia Crossroads 
```

Comment:
no number = no series

Maybe make this the publisher. Also needs to be in quotes.